content/en/news/releases/1.11.x/announcing-1.11/change-notes/index.md

```diff
@@ -0,0 +1,256 @@
+---
+title: Change Notes
+linktitle: 1.11.0
+subtitle: Minor Release
+description: Istio 1.11.0 release notes.
+publishdate: 2020-07-29
+release: 1.11.0
+weight: 10
+aliases:
+    - /news/announcing-1.11.0
+---
+
+{{< warning >}}
+This is an automatically generated rough draft of the release notes and has not yet been reviewed.
+{{< /warning >}}
+
+# Release Notes
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/about/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+
+
+## Traffic Management
+
+
+- **Improved** resolution of headless services via in-agent DNS to include endpoints
+from other clusters that are on the same network.
+  ([Issue #27342](https://github.com/istio/istio/issues/27342))
+
+- **Improved** usage of AUTO_PASSTHROUGH Gateways to no longer require configuring the `ROUTER_MODE` option on the gateway deployment; instead, it is automatically detected
+  ([Issue #33127](https://github.com/istio/istio/issues/33127))
+
+- **Improved** the XDS configuration generation to cache Envoy clusters, substantially improving performance of the control plane in many cases
+  ([Issue #26732](https://github.com/istio/istio/issues/26732))
+
+- **Improved** CNI network plugin to send logs to the CNI daemonset. This allows viewing CNI logs using `kubectl logs`, instead of looking at kubelet logs.
+  ([Issue #32437](https://github.com/istio/istio/issues/32437))
+
+- **Improved** service conflict resolution to favor Kubernetes Services over ServiceEntries with the same hostname.
+
+- **Updated** CNI install container and race condition repair container are combined into one container.
+  ([Issue #33712](https://github.com/istio/istio/issues/33712))
+
+- **Updated** the Istiod debug interface to be only accessible over localhost or with proper authenciation (mTLS or JWT).
+The recommended way to access the debug interface is through `istioctl experimental internal-debug`, which handles
+this automatically.
+  
+
+- **Added** a flag setting ShutdownDuration in pilot and its default value is 10s.
+  
+
+- **Added** an environment variable `PILOT_STATUS_UPDATE_INTERVAL` that is the interval to update the XDS distribution status and its default value is `500ms`.
+  
+
+- **Added** The Istio sidecar agent will offer HTTP endpoint localhost:15009/debug/<typeurl>.  GET requests
+to that URL will be resolved by sending an xDS discovery "event" to istiod.  This can be disabled by setting
+the following in the Istio Operator: meshConfig.defaultConfig.proxyMetadata.PROXY_XDS_DEBUG_VIA_AGENT=false
```

Comment:
Is this correct?  I don't find how to configure this via meshConfig:
https://preliminary.istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/

I was trying to figure out if this is configured only globally or per service.

cc @howardjohn @esnible

Reply:
It is correct. You can do it per-workload with annotation: https://preliminary.istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#ProxyConfig

Reply:
except I think the port number is wrong

Reply:
Ok, I see port should be 15004.  still don't see PROXY_XDS_DEBUG_VIA_AGENT or anything related to this in the proxyConfig link you sent.

Reply:
proxyMetadata is a map of arbitrary key value pairs that end up as env vars

Reply:
Ah!! thanks John!